roles/ara_api/tasks/install/pypi.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Install required packages from Pypi
+  pip:
+    name: "{{ pip_pkg }}"
+    virtualenv: "{{ ara_api_venv_path }}"
+    state: present
+    virtualenv_command: /usr/bin/python3 -m venv
+  loop:
+    - django
+    - django-cors-headers
+    - django-filter
+    - django-health-check
+    - django-utils
+    - djangorestframework
+    - dynaconf
+    - health-check
+    - pygments
+    - tzlocal
+    - whitenoise
+  loop_control:
+    loop_var: pip_pkg
```

Comment:
out of curiosity, is installing ``ara[server]`` not sufficient ? do the dependencies need to be installed individually ?

Reply:
You're right! I have updated the playbook to do just that :-)